addons/l10n_eg_edi_eta/__manifest__.py

```diff
@@ -10,7 +10,7 @@
     """,
     'author': 'Odoo S.A.',
     'website': 'https://www.odoo.com',
-    'category': 'account',
+    'category': 'Accounting/Accounting',
```

Comment:
This is the EDI module for the Egyptian accounting localization.
I guess `Localization` should be part of the category here. I mean, `l10n_fi_sale` is `Accounting/Localizations` and this is not?

Reply:
@lordkrandel updated the category to Accounting/Localizations/EDI